EIPS/eip-3529.md

```diff
@@ -3,7 +3,8 @@ eip: 3529
 title: Reduction in refunds
 author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
 discussions-to: https://ethereum-magicians.org/t/eip-3529-reduction-in-refunds-alternative-to-eip-3298-and-3403-that-better-preserves-existing-clearing-incentives/6097
-status: Draft
```

Comment:
I think technically this should have gone through the `Review` state first, but it was extensively discussed during 2 (or 3) ACD meetings.

@timbeiko potentially the rule of considering only EIPs in the `Review` )or `Last Call`) state for final inclusion decision could be more strictly enforced.

Reply:
It Technically okay to have skipped the step, but it shouldn't be the normal case. I remember writing something about what steps are allowed to be skipped, but I don't know it if ever got merged anywhere.

Being better with EIP-1 would help enforce some of these things as of now, what is and what isn't a rule isn't always clear.

Reply:
@axic fair enough. I think it's more important to move them to `Last Call` now though given that's what triggers folks to look at the EIPs, so I would rather go Draft -> Last Call directly vs. delaying it by a few days/weeks.